ethcore/Cargo.toml

```diff
@@ -32,6 +32,7 @@ ethjson = { path = "../json" }
 ethkey = { path = "../ethkey" }
 ethstore = { path = "../ethstore" }
 evmjit = { path = "../evmjit", optional = true }
+fetch = { path = "../util/fetch" }
```

Comment:
To reduce the number of dependencies IMHO we could re-export fetch from `price-info`.